### PR TITLE
ensure _questions array is set up early

### PR DESCRIPTION
**Pull request by moloko**
fixes https://github.com/adaptlearning/adapt_framework/issues/2971

fix various eslint issues

### CODE REVIEW

**Review by oliverfoster — APPROVED** [review]

File: js/adapt-assessmentArticleModel.js [hunk 1]
```diff
@@ -720,11 +716,7 @@ define([
         _score: score || 0
       });
 
-      if (score) {
-        scoreAsPercent = Math.round( score / maxScore  * 100);
-      } else {
-        scoreAsPercent = 0;
-      }
+      scoreAsPercent = score ? Math.round(score / maxScore * 100) : 0;
```
Comment (tomgreenfield) [inline js/adapt-assessmentArticleModel.js:719]: This can be done on variable creation on line 701.

**Review by tomgreenfield — APPROVED** [review]
:eye:

**Review by tomgreenfield — APPROVED** [review]
:eye:

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Comment by moloko** [comment]
@oliverfoster do you want to check again?

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**